christfellowshipchurch/web-app-v2
# Fixed Group End Date Bug

### About
This PR updates the `GroupDateTime` component by adding a catch for groups that do not have a start/end date because the group has finished.

### Test Instructions
* go to a group that has ended or the end date is today and see that it loads correctly on web.

### Screenshots
<img width="1295" alt="image" src="https://github.com/christfellowshipchurch/web-app-v2/assets/46049974/ec9d08dd-ec7e-4c49-8e38-c5c8ae5b8765">

### Closes Tickets
[CFDP-2130](https://christfellowshipchurch.atlassian.net/browse/CFDP-2130)

[CFDP-2130]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
